tests/cylc-cat-state/basic/state-check.py

```diff
@@ -62,18 +63,20 @@ def main(argv):
                 while next:
                     res.append(next[0])
                     next = cur.fetchmany()
-            except:
+            except:  # noqa: E722
```

Comment:
... seems a bit pointless modifying test code too, but I suppose there's no harm in doing so.